packages/api-explorer/src/components/ViewOnlyMode/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React, {useEffect} from 'react'
+import PropTypes from 'prop-types'
+
+import docPropTypes from '../../prop-types/doc'
+import colors from '../../colors'
+
+const REDOC_OPTIONS = {
+  disableSearch: true,
+  hideDownloadButton: true,
+  lazyRendering: true,
+  theme: {
+    spacing: {
+      sectionHorizontal: 20,
+      sectionVertical: 10
+    }
+  }
+}
+
+const REDOC_CONTAINER_ID = 'redoc-container'
+
+const SUPPORTED_METHODS = Object.keys(colors).filter(key => colors[key].color)
+
+const changeApiMethodColor = (apiContent) => {
+  SUPPORTED_METHODS
+    .map(method => ({method, spanElement: apiContent.querySelector(`span[type=${method}]`)}))
+    .filter(({spanElement}) => spanElement)
+    .forEach(({method, spanElement}) => {
+      const currentColor = colors[method];
+      spanElement.style.backgroundColor = currentColor.color
+      spanElement.style.border = currentColor.border
+    })
+}
+
+const reduceHeight = (apiContent) => {
+  const lastSection = [...apiContent.querySelectorAll('[data-section-id]')].pop()
+  lastSection.style.minHeight = 'unset'
+}
+
+const patchRedoc = () => {
+  const redocMenu = document.querySelector(".menu-content")
+  if (redocMenu) {
+    const apiContent = redocMenu.parentElement.querySelector(".api-content")
```

Comment:
If api-content not exists?

Reply:
IMO an error should be raised, because it means that Redoc has not be loaded correctly

Reply:
An html error?

Reply:
yep, an error inside the tab.
If you prefer, we can handle, avoiding any kind of action: btw, if this happen, it means that the content of the tab is empty.